### PR TITLE
Sofia/navigating around

### PR DESCRIPTION
**Pull request by sofiiako**

### CODE REVIEW

File: Introduction/Getting started/Course View/task-info.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+type: theory
+files:
+- name: src/main.rs
+  visible: true
+- name: Cargo.toml
+  visible: true
+- name: edu_course_view.png
+  visible: true
```
Comment (Geravant) [inline Introduction/Getting started/Course View/task-info.yaml:8]: Let's hide it

File: Introduction/Getting started/Editor/task-info.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+type: theory
+files:
+- name: src/main.rs
+  visible: true
+- name: Cargo.toml
+  visible: true
+- name: edu_context_menu_run.png
+  visible: true
```
Comment (Geravant) [inline Introduction/Getting started/Editor/task-info.yaml:8]: Let's hide it

File: Introduction/Getting started/Editor/task-info.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+- name: edu_editor.png
+  visible: true
```
Comment (Geravant) [inline Introduction/Getting started/Editor/task-info.yaml:10]: Let's hide it

File: Introduction/Getting started/Navigating Around/task-info.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+type: theory
+files:
+- name: src/main.rs
+  visible: true
+- name: Cargo.toml
+  visible: true
+- name: edu_course_overview.png
```
Comment (Geravant) [inline Introduction/Getting started/Navigating Around/task-info.yaml:7]: Let's hide it

File: Introduction/Getting started/Task Description/task-info.yaml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+files:
+- name: src/main.rs
+  visible: true
+- name: Cargo.toml
+  visible: true
+- name: images/back.svg
+  visible: true
+- name: images/commentTask.svg
+  visible: true
+- name: images/edu_next_button.png
```
Comment (Geravant) [inline Introduction/Getting started/Task Description/task-info.yaml:11]: Let's hide it

File: Introduction/Getting started/Task Description/task-info.yaml [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+  visible: true
+- name: images/edu_peek_solution.png
+  visible: true
```
Comment (Geravant) [inline Introduction/Getting started/Task Description/task-info.yaml:14]: Let's hide it

File: Introduction/Getting started/Task Description/task-info.yaml [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+- name: images/edu_task_description_window_settings.png
```
Comment (Geravant) [inline Introduction/Getting started/Task Description/task-info.yaml:15]: Let's hide it

File: Introduction/Getting started/Task Description/task-info.yaml [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+  visible: true
+- name: images/forward.svg
```
Comment (Geravant) [inline Introduction/Getting started/Task Description/task-info.yaml:17]: Let's hide it

File: Introduction/Getting started/Task Description/task-info.yaml [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+  visible: true
+- name: images/gear.svg
```
Comment (Geravant) [inline Introduction/Getting started/Task Description/task-info.yaml:19]: Let's hide it

File: Introduction/Getting started/Task Description/task-info.yaml [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+  visible: true
+- name: images/hideToolWindow.svg
```
Comment (Geravant) [inline Introduction/Getting started/Task Description/task-info.yaml:21]: Let's hide it

File: Introduction/Getting started/Task Description/task-info.yaml [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+  visible: true
+- name: images/reset.svg
+  visible: true
```
Comment (Geravant) [inline Introduction/Getting started/Task Description/task-info.yaml:24]: Let's hide it

**Review by Geravant — CHANGES_REQUESTED** [review]
Everything looks very cool, screen captures are much better than in the scala
Let's just hide them from the learner

**Comment by sofiiako** [comment]
@Geravant changes made! :)

**Review by Geravant — APPROVED** [review]

**Pull request merged**